oio/event/filters/xcute.py

```diff
@@ -0,0 +1,38 @@
+# Copyright (C) 2019 OpenIO SAS, as part of OpenIO SDS
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+from oio.event.evob import Event
+from oio.event.filters.base import Filter
+from oio.xcute.common.worker import XcuteWorker
+
+
+class XcuteFilter(Filter):
+
+    def init(self):
```

Comment:
Isn't it missing args here? Namely `app, conf` as used below?

Reply:
No, it isn't missing args. It's `init`, not `__init__`.